test/site_selection.jl

```diff
@@ -28,3 +29,30 @@ end
     @test all([in(sid, [2, 3]) for sid in prefseedsites[prefseedsites.>0]])
     @test all([in(sid, [2, 3]) for sid in prefshadesites[prefshadesites.>0]])
 end
+
+@testset "Guided site selection without ADRIA ecological model" begin
+    dom = ADRIA.load_domain(joinpath(here, "Example_domain"))
+
+    criteria_df = ADRIA.sample(ex_domain, 5) # get scenario dataframe
+
+    area_to_seed = 1.5 * 10^-6 # area of seeded corals in km^2
+    ts = 5 # time step to perform site selection at
+
+    sumcover = 0.1 .* ones(5, size(dom.site_data, 1))
+    ranks = run_site_selection(dom, criteria_df[criteria_df.guided.>0, :], sumcover, area_to_seed, ts)
+
+    # Check that only 4 sites make it through depth and heat/wave risk filter  
+    for nscen in 1:size(ranks, 1)
+        @test all(ranks["scen_$nscen", 5, :] .== 0.0) || "Sites which should have been filtered in scenario $nscen have still been ranked."
+    end
+
+    @test size(ranks, 1) == sum(criteria_df.guided .> 0) || "Specified number of scenarios was not carried out."
+
+    sel_sites = unique(ranks)
+    sel_sites = sel_sites[sel_sites.!=0.0]
+    site_ids = collect(Float64, 1:size(dom.site_data, 1))
```

Comment:
Should this be `Int64` instead of `Float64`?

Reply:
this is used to compare to the unique values in ranks, and ranks are output as Floats at the moment, but this comment mkes me realise I should adjust this test because they aren't site ids ...